src/commands/deploy.ts

```diff
@@ -119,27 +118,6 @@ export async function deployCommand(options: GenezioDeployOptions) {
             throw error;
         });
 
-        // Enable cloud provider AB Testing
-        // This is ONLY done in the dev environment and if DISABLE_AB_TESTING is not set.
-        if (
-            ENVIRONMENT === "dev" &&
-            configuration.backend &&
-            process.env["DISABLE_AB_TESTING"] !== "true"
-        ) {
-            const yamlConfig = await configIOController.read(/* fillDefaults= */ false);
-            if (!yamlConfig.backend) {
-                throw new UserError("No backend entry in genezio configuration file.");
-            }
-            yamlConfig.backend.cloudProvider = await performCloudProviderABTesting(
```

Comment:
Should we also remove `performCloudProviderABTesting` and tests associated to it (`tests/performAbTesting.test.ts`)?